### ISSUE TITLE
Sri Lanka: Historic data

### ISSUE THREAD

**Issue by tmtmtmtm** [everypolitician/everypolitician-data#889]
http://www.parliament.lk/en/members-of-parliament/directory-of-past-members/?cletter=A

**Comment by Abbie2020** [comment]
I'm working on this.

**Comment by Abbie2020** [comment]
https://morph.io/Abbie2020/sri_lanka_past_members_of_parliament

**Comment by tmtmtmtm** [comment]
Hi @Abbie2020 

Thanks for this. There doesn't seem to be a term column, though. We need to know when each person was actually in the Parliament to be able to use the information…

**Comment by Abbie2020** [comment]
The Parliament website doesn't seem to have this information. Would it be possible to combine this data with the Wikipedia data to get the terms?

**Comment by tmtmtmtm** [comment]
Hmmm. That seems rather non-useful on their part! The individual pages do give a _little_ more info, in that they do sometimes show party changes over time in the "Political Career" section (e.g. http://www.parliament.lk/en/members-of-parliament/directory-of-past-members/viewMember/1463), but that looks completely inconsistent. So, yes, I suspect it's better using another source (e.g. Wikipedia) for the membership information and then using this just to add some basic biographical info[1] + photos for those.


[1] Though it looks like you're losing the second half of people's names…

**Comment by Abbie2020** [comment]
Ah crap! Think it might be the commas... Yeah I did consider the 'Political Career' section but, as you say, it's a bit random. Will have another look at the names.

**Comment by Abbie2020** [comment]
@tmtmtmtm Names should look better now. https://morph.io/Abbie2020/sri_lanka_past_members_of_parliament

**Comment by tmtmtmtm** [comment]
Great — thanks. We still can't actually _do_ anything with these until we scrape another source that includes the term information, but at least this can be matched against that once that happens.

**Comment by Abbie2020** [comment]
Yeah I know, thought I might as well do it as it was so similar to the other one.
